test/gtest/uct/uct_test.cc

```diff
@@ -159,17 +160,32 @@ uct_test::uct_test() {
         status = uct_md_iface_config_read(md, NULL, NULL, NULL, &m_iface_config);
     } else if (!strcmp(GetParam()->tl_name.c_str(), "sockaddr")) {
         m_iface_config = NULL;
-        return;
     } else {
         status = uct_md_iface_config_read(md, GetParam()->tl_name.c_str(), NULL,
                                           NULL, &m_iface_config);
     }
 
     ASSERT_UCS_OK(status);
     uct_md_close(md);
+
+    component_attr.field_mask = UCT_COMPONENT_ATTR_FIELD_NAME |
+                                UCT_COMPONENT_ATTR_FIELD_FLAGS;
+    /* coverity[var_deref_model] */
```

Comment:
what was the cov issue?

Reply:
@yosefe 
`Event var_deref_model: 	Passing "&component_attr" to "uct_component_query", which dereferences null "component_attr.md_resources"`
same as here for example - 
https://github.com/openucx/ucx/blob/master/test/gtest/uct/uct_test.cc#L114